src/index.ts

```diff
@@ -267,30 +328,120 @@ export function withdrawalTransaction(
     sequence: timelock,
   });
 
+  // withdraw tx always has 1 output only
+  const estimatedFee = getEstimatedFee(feeRate, psbt.txInputs.length, 1);
+
   psbt.addOutput({
     address: withdrawalAddress,
-    value: tx.outs[outputIndex].value - withdrawalFee,
+    value: tx.outs[outputIndex].value - estimatedFee,
   });
 
-  return psbt;
+  return {
+    psbt,
+    fee: estimatedFee
+  };
+}
+
+// Slashing of the staking transaction when no unbonding has been performed
+// it spends the staking output of the staking transaction
+// Outputs:
+//   - The first one sends input * slashing_rate funds to the slashing address
+//   - The second one sends input * (1-slashing_rate) - fee funds back to the user’s address
+export function slashingNoUnbondingTransaction(
+  scripts: {
+    slashingScript: Buffer,
+    timelockScript: Buffer,
+    unbondingScript: Buffer,
+    unbondingTimelockScript: Buffer,
+  },
+  transaction: Transaction,
```

Comment:
```suggestion
  stakingTransaction: Transaction,
```
We can be explicit since we know with what the function should be called for.